### PR TITLE
Add Bun logo in README

### PR DESCRIPTION
**Pull request by DanielTolentino**
Add Bun logo at the top of README
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 # bun
+<p align="center">
```
Comment (toastal) [inline README.md:2]: Prefer: `[](https://bun.sh)`. 

* Adding HTML to the Markdown ruins the lightweight syntax and centering it adds nothing but noise to the plaintext
* `alt="Logo"` is not helping assistive technology. The logo is purely decorative so it it should be empty (e.g. `alt=""`)

**Comment by toastal** [comment]
Prefer: `[](https://bun.sh)`. 

* Adding HTML to the Markdown ruins the lightweight syntax and centering it adds nothing but noise to the plaintext
* `alt="Logo"` is not helping assistive technology. The logo is purely decorative so it it should be empty (e.g. `alt=""`)

File: README.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
...
+  <a href="https://bun.sh"><img src="https://bun.sh/logo@2x.png" alt="Logo"></a>
```
Comment (toastal) [inline README.md:3]: Prefer: `[](https://bun.sh)`

* Adding HTML to the Markdown ruins the lightweight syntax and centering it adds nothing but noise to the plaintext and nothing useful to the rendered output (bad trend).
* `alt="Logo"` is not helping assistive technology or users with images disabled. The logo is purely decorative so it should be empty (e.g. `alt=""`).

**Comment by Jarred-Sumner** [comment]
Thanks.

I will merge this and then remove the `alt` label per @toastal's comment (I like the centering though)

**Comment by toastal** [comment]
`alt` wasn't removed

**Pull request merged**